examples/heatmap-earthquakes.js

```diff
@@ -9,33 +9,37 @@ import VectorSource from '../src/ol/source/Vector.js';
 const blur = document.getElementById('blur');
 const radius = document.getElementById('radius');
 
-const vector = new HeatmapLayer({
+const heatmap = new HeatmapLayer({
   source: new VectorSource({
     url: 'data/kml/2012_Earthquakes_Mag5.kml',
     format: new KML({
       extractStyles: false,
     }),
   }),
-  blur: parseInt(blur.value, 10),
-  radius: parseInt(radius.value, 10),
+  blur: ['var', 'blur'],
+  radius: ['var', 'radius'],
+  variables: {
+    blur: parseInt(blur.value, 10),
+    radius: parseInt(radius.value, 10),
+  },
```

Comment:
For this example, the direct use of blur / setBlur() and radius / setRadius() seems more straightforward, so I think this should be changed back.

Reply:
Indeed, I see what you mean. The reason I changed this example is that now `setRadius()` will trigger a recreation of the renderer, which makes the layer flicker. Using style variables instead will allow for a smoother change of radius or blur (like before this PR), but I agree that it makes the code a little bit less straightforward.

I'll change it back and then you can try it again to see what I mean.

Reply:
I see. Wouldn't it make sense then to change things so internally style variables are used, and `setRadius()` updates them?

Reply:
That would be possible, but keep in mind that `setRadius()` can also take expressions now. So we'd need to have two code branches: if it's an integer, update the style variable; otherwise, recreate the renderer with the new expression.

I assumed that `setRadius` and `setBlur` wouldn't typically be called continuously like in the example and thus that it would be acceptable to have this flicker when called, but maybe my assumption is wrong? In that case, I can indeed find a way to make it work with style variables internally.

Reply:
It might be worth having `setRadius()` and `setBlur()` distinguish these two code paths, for user convenience.